certchain/certmanager/multicert_disk_cache.go

```diff
@@ -113,9 +113,13 @@ func (d *MultiCertDiskCache) Read(digest string) (*certchain.AugmentedChain, err
 	cert := path.Join(d.CertDir, digest+".pem")
 	ocsp := path.Join(d.CertDir, digest+".ocsp")
 	rawChain, err := certchainutil.ReadRawChainFile(cert)
-	errs = multierror.Append(errs, err)
+	if err != nil {
+		errs = multierror.Append(errs, ErrNotFound)
```

Comment:
Is there a way we can attach the original error (returned by ReadRawChainFile), too? Could contain useful info for the logs. e.g. file not found vs no read permissions. I'm not sure if that should be something like errors.Wrap or just attach both errors using multierror; it's been a while since I coded any webpkg.

Reply:
This is a good suggestion.  I attached both errors.